### ISSUE TITLE
[Bug]: Fatal error detected by the Java Runtime Environment

### ISSUE THREAD

**Issue by kaosCloud** [Stirling-Tools/Stirling-PDF#2569]
### Installation Method

Docker

### The Problem

After installing Stirling on Docker in macOS 15.2 (Mac Mini M4), I cannot start the container with error as below. Both docker and Stirling are at the newest version.

### Version of Stirling-PDF

0.36.5

### Relevant Log Output

```txt
2024-12-29 22:05:12 Copying original files without overwriting existing files
2024-12-29 22:05:12 Running Stirling PDF with DOCKER_ENABLE_SECURITY=false and VERSION_TAG=0.36.5
2024-12-29 22:05:12 Installing font package: font-noto
2024-12-29 22:05:12 fetch https://dl-cdn.alpinelinux.org/alpine/v3.20/main/aarch64/APKINDEX.tar.gz
2024-12-29 22:05:12 fetch https://dl-cdn.alpinelinux.org/alpine/v3.20/community/aarch64/APKINDEX.tar.gz
2024-12-29 22:05:12 fetch https://dl-cdn.alpinelinux.org/alpine/edge/main/aarch64/APKINDEX.tar.gz
2024-12-29 22:05:12 fetch https://dl-cdn.alpinelinux.org/alpine/edge/community/aarch64/APKINDEX.tar.gz
2024-12-29 22:05:13 fetch https://dl-cdn.alpinelinux.org/alpine/edge/testing/aarch64/APKINDEX.tar.gz
2024-12-29 22:05:13 OK: 1159 MiB in 369 packages
2024-12-29 22:05:13 Setting permissions and ownership for necessary directories...
2024-12-29 22:05:13 #
2024-12-29 22:05:13 # A fatal error has been detected by the Java Runtime Environment:
2024-12-29 22:05:13 #
2024-12-29 22:05:13 #  SIGILL (0x4) at pc=0x0000ffff7753fc5c, pid=10, tid=20
2024-12-29 22:05:13 #
2024-12-29 22:05:13 # JRE version:  (21.0.5+11) (build )
2024-12-29 22:05:13 # Java VM: OpenJDK 64-Bit Server VM (21.0.5+11-alpine-r0, mixed mode, sharing, tiered, compressed oops, compressed class ptrs, g1 gc, linux-aarch64)
2024-12-29 22:05:13 # Problematic frame:
2024-12-29 22:05:13 # j  java.lang.System.registerNatives()V+0 java.base@21.0.5
2024-12-29 22:05:13 #
2024-12-29 22:05:13 # No core dump will be written. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
2024-12-29 22:05:08 Picked up JAVA_TOOL_OPTIONS:  -XX:MaxRAMPercentage=75
2024-12-29 22:05:08 /scripts/init.sh: line 31:     9 Aborted                 /scripts/init-without-ocr.sh "$@"
2024-12-29 22:05:13 Picked up JAVA_TOOL_OPTIONS:  -XX:MaxRAMPercentage=75
2024-12-29 22:05:13 /scripts/init.sh: line 31:    10 Aborted                 /scripts/init-without-ocr.sh "$@"
2024-12-29 22:05:13 #
2024-12-29 22:05:13 # An error report file with more information is saved as:
2024-12-29 22:05:13 # /tmp/hs_err_pid10.log
2024-12-29 22:05:13 [0.007s][warning][os] Loading hsdis library failed
2024-12-29 22:05:13 #
2024-12-29 22:05:13 # The crash happened outside the Java Virtual Machine in native code.
2024-12-29 22:05:13 # See problematic frame for where to report the bug.
2024-12-29 22:05:13 #
```

### No Duplicate of the Issue

- [X] I have verified that there are no existing issues raised related to my problem.

**Comment by Frooodle** [comment]
Seems like it might have downloaded the non arm version docker at a guess... I'd try validate if you are using the arm version 
Other than that not sure

**Comment by kaosCloud** [comment]
Turns out, it's JDK/Docker bug, workaround is to set JAVA_TOOL_OPTIONS environmental to -XX:UseSVE=0, it resolved issue for me.
